NameMatching.Tests/NameTests.cs

```diff
@@ -2,38 +2,16 @@
 
 public class NameTests
 {
-	//[Fact]
-	// public void Constructor_FullName_ShouldParseCorrectly()
-	// {
-	// 	var name = new Name("John Adam Smith Jr.");
-	//
-	// 	name.FirstName.Should().Be("John");
-	// 	name.MiddleName.Should().Be("Adam");
-	// 	name.LastName.First().Should().Be("Smith");
-	// 	name.Suffix.Should().Be("Jr.");
-	// }
-
-	[Fact]
-	public void GetFullName_ShouldReturnCorrectFullName()
-	{
-		var name = new Name("John", "Adam", "Smith", "Jr.");
-		var fullNameWithSuffix = name.GetFullName(includeSuffix: true);
-		var fullNameWithoutSuffix = name.GetFullName(includeSuffix: false);
-
-		fullNameWithSuffix.Should().Be("John Adam Smith Jr.");
-		fullNameWithoutSuffix.Should().Be("John Adam Smith");
-	}
-
 	[Fact]
 	public void GetTokenizedName_ShouldReturnCorrectTokens()
 	{
 		var name = new Name("John", "Adam", "Smith", "Jr.");
 		var tokensWithSuffix = name.GetTokenizedName(includeSuffix: true);
 		var tokensWithoutSuffix = name.GetTokenizedName(includeSuffix: false);
 
-		tokensWithSuffix.Should().Equal(new List<string> { "John", "Adam", "Smith", "Jr." });
+		tokensWithSuffix.Should().Equal(new List<string> { "John", "Adam", "Smith", "jr" });
 		tokensWithoutSuffix.Should().Equal(new List<string> { "John", "Adam", "Smith" });
-		tokensWithSuffix.Should().Equal(new List<string> { "John", "Adam", "Smith", "Jr." });
+		tokensWithSuffix.Should().Equal(new List<string> { "John", "Adam", "Smith", "jr" });
```

Comment:
Why not mix in some case differences in the other name parts, too? john ADAm smiTh

Reply:
I need to still add additional unit tests to cover new scenarios, just didn't add them to make sure you're unblocked.